content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -49,10 +49,10 @@ kind: ClusterRole
 metadata:
   name: prometheus
 rules:
-  - nonResourceURLs:
-      - "/metrics"
-    verbs:
-      - get
+- nonResourceURLs:
+  - "/metrics"
+  verbs:
+  - get
```

Comment:
It is a personal taste/preference to indent list items. In this example, when we indent the `- nonResourceURLs` on line 52, the visual effect is evident that `rules` is a list and the `nonResourceURLs` is an attribute of the first list item.
However, if we remove that indentation, the syntax is still correct, but since the `-nonResourceURLs` line has the same indentation as its parent (`rules`), it is unclear that `rules` is a list.
All YAML processors can parse both syntax correctly. The only difference is about human readability.

With that, I'm against this change.